packages/prices/utils/price.ts

```diff
@@ -134,7 +180,23 @@ export const formatPrice = (
 	}
 
 	const currency: Currency = getCurrency( currencyData );
-	const formattedPrice: number = priceInt / 10 ** currency.minorUnit;
+
+	const integerValue: string = getIntegerValue(
+		priceInt,
+		currency.thousandSeparator,
+		currency.minorUnit
+	);
+
+	const decimalValue: string | undefined = getDecimalValue(
+		priceInt,
+		currency.minorUnit
+	);
+
+	const formattedPrice: string =
+		currency.minorUnit > 0
+			? integerValue + currency.decimalSeparator + decimalValue
```

Comment:
`decimalValue` can be `undefined`, so I think we should check for its value before appending it (and the decimal separator) for extra safety.

Reply:
Thanks for pointing this out, @dinhtungdu. I added checks both for `currency.decimalSeparator` and `decimalValue` before using them.